basket/news/tests/test_tasks.py

```diff
@@ -771,3 +787,19 @@ def test_timestamps_nothin_cached(self, cache_mock, sfmc_mock):
             'FXA_ID': data['uid'],
             'NewEmailAddress': data['email'],
         })
+
+
+class GmttimeTests(TestCase):
+    def test_basetime_provided(self):
+        basetime = parse_date('2018-05-18T13:21:54.188Z')
+        formatted_time = gmttime(basetime)
+        self.assertEqual(formatted_time, 'Fri, 18 May 2018 21:31:54 GMT')
+
+    def test_basetime_not_provided(self):
```

Comment:
@pmac - does this test provide any value? The fact that the test could fail makes me nervous.

Reply:
I think the way to go here would be to mock `datetime.datetime.now`, create a datetime object as the `return_value` of the mock, then call `gmttime()` and assert that the mock was called and that the formatted time is what is expected based on your datetime object. That's the only way I can see for making this test both worthwhile and stable.